wordpress-mobile/WordPress-Android
# Issue/5310 premium themes link

Fixes #5310 by adding a premium themes link at the bottom of the screen (consistently with the Plans section)

![screenshot_20170306-160334](https://cloud.githubusercontent.com/assets/6597771/23655769/3c2bf1b4-0315-11e7-9c8d-b974c6e08efe.png)


To test:

1. open the Themes section in the main tab
2. tap on the search bar
3. the blue bottom button is shown, and you can explore premium themes by tapping on it (an external browser is opened)
